debug_dataframe_size.py

```
@@ -0,0 +1,77 @@
+
+import os
+import shutil
+import tempfile
+
+import pandas as pd
+from openpyxl import Workbook
+
+from sphinxcontrib.jsontable.core.excel_reader import ExcelReader
+
+
+def main():
```

Pylint (3.3.7)

[refactor] Too many local variables (16/15)

(R0914)